documentation/usage/gradle/index.md

```diff
@@ -176,6 +176,32 @@ flyway {
 
 For details on how to setup and use custom Gradle configurations, see the [official Gradle documentation](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ConfigurationContainer.html).
 
+### Using Flyway Database Types
+
+For some Flyway database types (non core), like [Cloud Spanner](/documentation/database/cloud-spanner) or [SQL Server](/documentation/database/sqlserver), you'll need to actually use a `buildScript` to get your
+Gradle commands to work properly. You need to enable putting the database type on the build classpath, and not
+the project classpath.
```

Comment:
Could you reword this (keeping the links) to: ``For some Flyway database types, like Cloud Spanner and SQL Server you'll need to add a dependency to the database type in a `buildscript` closure to get your Gradle commands to work properly. This puts the database type on the build classpath, and not the project classpath.``

It's a bit more succinct and removes the `non core` component which isn't strictly true - we are looking at a long-term goal of having a module per database type in order to reduce the bloat in Flyway Core

Reply:
Yeah, fwiw, I'd be _awesome_ to have the error message itself spit out _what_ database types _are_ installed and ready to go. Fighting with Gradle to see the classpath is :rage4: imo

(And happy to make the change 😌 )

Reply:
Could you write that suggestion up as an issue [here](https://github.com/flyway/flyway/issues) in order to track it